stixcore/products/level0/scienceL0.py

```diff
@@ -99,24 +99,26 @@ def parent(self):
         return np.unique(self.control['parent'])
 
     def __add__(self, other):
-        # if (np.all(self.control == other.control) and self.scet_timerange == other.scet_timerange
-        #         and len(self.data) == len(other.data)):
-        #     return self
-        # combined_control_index = other.control['index'] + self.control['index'].max() + 1
-        # control = vstack((self.control, other.control))
-        # cnames = control.colnames
-        # cnames.remove('index')
-        # control = unique(control, cnames)
-        #
-        # combined_data_index = other.data['control_index'] + self.control['index'].max() + 1
-        # data = vstack((self.data, other.data))
-        #
-        # data_ind = np.isin(combined_data_index, combined_control_index)
-        # data = data[data_ind]
-        #
-        # return type(self)(service_type=self.service_type, service_subtype=self.service_subtype,
-        #                   ssid=self.ssid, data=data, control=control)
-        raise(ValueError(f"Tried to combine 2 BSD products: {self} and {other}"))
+        if self.complete or other.complete:
+            raise(ValueError(f"Tried to combine 2 BSD products: \n{self} and \n{other}"))
+
+        if (np.all(self.control == other.control) and self.scet_timerange == other.scet_timerange
+                and len(self.data) == len(other.data)):
+            return self
+        combined_control_index = other.control['index'] + self.control['index'].max() + 1
```

Comment:
This seems to assume other and self are ordered what if `other.control['index'] = [4, 5, 6]` and `self['index'] = [1, 2, 3]` for example?

Reply:
is removed